src/api/app/assets/javascripts/webui2/project.js

```diff
@@ -0,0 +1,23 @@
+function renderProjectsTable(length) { // jshint ignore:line
+  length = (typeof length === "undefined") ? 25 : length;
```

Comment:
Please use a [default parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) instead.

Reply:
Changing the code to

```function renderProjectsTable(length = 25)```

raises this error:

```Unexpected token operator «=», expected punc «,». To use ES6 syntax, harmony mode must be enabled with Uglifier.new(:harmony => true)```

I think it is better to keep it like it is for the moment. Perhaps we will refactor this weird Javascript in the future.

Reply:
hmm, so uglifier enforces to use old JS? Where would we set this harmony option?

Reply:
@coolo:

```
config.assets.js_compressor = Uglifier.new(harmony: true)
```

This typeof is really ugly... I would use `||` similarly as in ruby:

`length = length || 25`

Reply:
> hmm, so uglifier enforces to use old JS? Where would we set this harmony option?

The problem is that Internet Explorer is not supporting ES6 as it seems. Unless we use a transpiler we have to stick with the old syntax.